testfiles/1ZoneDataCenterCRAC_wApproachTemp_ClassH1.idf

```diff
@@ -580,8 +580,8 @@
     26.55556,                !- <none>
     29.33333,                !- <none>
     32.11111,                !- <none>
-    34.88889,                !- Extended Field
-    50.00000;                !- Extended Field
+    34.88889,                !- <none>
+    50.00000;                !- <none>
```

Comment:
I went a head and ran IDF formatting again with these changes, and this is the result. Cleaning up this can be a follow-up issue later.